### PR TITLE
deploy app to streamlit dashboard

### PR DESCRIPTION
**Pull request by piotrm0**
Prototype only. Can release as "experimental". See https://github.com/truera/trulens/blob/bc97162a3198df08a755793e44aa30756e293f53/trulens_eval/examples/experimental/streamlit_appui_example.ipynb for how to use. Also overview videos:

https://www.loom.com/share/d1c204f98463496bae1e5ce048219d05
https://www.loom.com/share/466d71f0eb8e422c9b882481b26d7b5d

### CODE REVIEW

File: trulens_eval/examples/experimental/appui_example.ipynb [hunk 1]
```diff
@@ -42,6 +42,13 @@
     ")"
```
Comment (rshih32) [inline trulens_eval/examples/experimental/appui_example.ipynb:42]: <pre class="ql-syntax" spellcheck="false">Line #30.    aui.d
</pre><p>Why do we want to call this with an attribute simply the letter <code>d</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: trulens_eval/examples/experimental/streamlit_appui_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,143 @@
+{
```
Comment (rshih32) [inline trulens_eval/examples/experimental/streamlit_appui_example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #21.        _dev=Path().cwd().parent.parent.resolve()
</pre><p>nit: dev code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (rshih32) [inline trulens_eval/examples/experimental/streamlit_appui_example.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #32.        initial_app_loader=load_langchain_app
</pre><p>Maybe add the comment about needing blank memory as the reason here in the notebook so people setting it up know why its done this way.</p><p>Also; is memory.clear() something that can be utilized instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/469/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (piotrm0) [reply trulens_eval/examples/experimental/streamlit_appui_example.ipynb:1]: Can we keep experimental examples in dev state and then make non-experimental notebooks when they mature?

Comment (piotrm0) [reply trulens_eval/examples/experimental/streamlit_appui_example.ipynb:1]: Added notes to explain why that is insufficient.

File: trulens_eval/examples/experimental/appui_example.ipynb (hunk 1 above)
Comment (piotrm0) [reply trulens_eval/examples/experimental/appui_example.ipynb:42]: Renamed to "widget" instead.

File: trulens_eval/examples/experimental/streamlit_appui_example.ipynb (hunk 2 above)
Comment (rshih32) [reply trulens_eval/examples/experimental/streamlit_appui_example.ipynb:1]: sure that makes sense

**Pull request merged**